rc5/CHANGELOG.md

```diff
@@ -7,3 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.0.1 (2023-02-10)
 - Initial release
+
+## 0.1.0 (2023-08-30)
+- Added u8 and u128 word size support.
+- Enabled custom word size, key size, and round count values.
+- Deprecated old predefined RC5 cipher types.
```

Comment:
We put newest versions first in changelogs, and also generally separate release PRs from code change PRs.

```suggestion
## Unreleased
- Added u8 and u128 word size support.
- Enabled custom word size, key size, and round count values.
- Deprecated old predefined RC5 cipher types.

## 0.0.1 (2023-02-10)
- Initial release
```